src/transforms/transform-pull-request.test.ts

```diff
@@ -134,4 +134,88 @@ describe("pull_request transform", () => {
 			updateSequenceId: 12345678
 		});
 	});
+
+	it("should not contain createPullRequestUrl on the payload if length > 2000", async () => {
+		const pullRequestList = Object.assign({},
+			transformPullRequestList
+		);
+
+		const fixture = pullRequestList[2];
+		fixture.title = "[TEST-0] Branch payload with loads of issue keys Test";
+
+		mockSystemTime(12345678);
+
+		githubNock.get(`/users/${fixture.user.login}`)
+			.reply(200, {
+				...fixture.user,
+				name: "Some User Name"
+			});
+
+		githubNock.get(`/users/${fixture.head.user.login}`)
+			.reply(200, {
+				...fixture.head.user,
+				name: "Last Commit User Name"
+			});
+
+		const data = await transformPullRequest(GitHubAPI(), fixture as any);
+
+		const { updated_at, title } = fixture;
+
+		const issueKeys = Array.from(new Array(250)).map((_, i) => `TEST-${i}`);
+
+		expect(data).toMatchObject({
+			id: 100403908,
+			name: "integrations/test",
+			pullRequests: [
+				{
+					author: {
+						avatar: "https://avatars0.githubusercontent.com/u/173?v=4",
+						name: "Some User Name",
+						url: "https://api.github.com/users/bkeepers"
+					},
+					destinationBranch: "devel",
+					destinationBranchUrl: "https://github.com/integrations/test/tree/devel",
+					displayId: "#51",
+					id: 51,
+					issueKeys,
+					lastUpdate: updated_at,
+					sourceBranch: "use-the-force",
+					sourceBranchUrl:
+						"https://github.com/integrations/test/tree/use-the-force",
+					status: "OPEN",
+					timestamp: updated_at,
+					title: title,
+					url: "https://github.com/integrations/test/pull/51",
+					updateSequenceId: 12345678
+				}
+			],
+			branches: [
+				{
```

Comment:
So for other scenarios where generateCreatePullRequestUrl < 2000 (example of test above), we would have a `createPullRequestUrl` parameter here. Since in this test url > 2000, that field is not going in the payload anymore.